src/core/utils/helpers/url.ts

```diff
@@ -134,8 +134,8 @@ export function redirectToLoginAndBack({
   returnUrl,
   trackingFunction
 }: RedirectToLoginAndBackParams) {
-  const { pathname, search } = returnUrl;
-  const localPathToReturnTo = `${pathname}${search}`;
+  const { pathname, search, hash } = returnUrl;
+  const localPathToReturnTo = `${pathname}${search}${hash || ""}`;
```

Comment:
My ts server in vs code says that `hash` is a string. Is it necessary to write: `hash || ""`. Isn't `hash` sufficient?

Reply:
True. We now output just hash :)